code/src/components/Artist.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+
+export const Artist = (props) => {
+    return (
+        <div className='artist-name'>
+            {
+                props.item.artists.map((artist) => {
+                    return <a 
+                            key={artist.id}
+                            href={artist.external_urls.spotify}>
```

Comment:
Wrong indents, you seem to forgot about parentheses which can wrap returned component